### DISCUSSION TITLE
Memperbesar Leaflet

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#260] [Q&A] [answered]
Hi ,

Bagaimana caranya untuk menambah **height** pada leaflet seperti pada contoh yang diberikan? Karena leaflet saya agak gepeng dan setelah diutak atik **height** nya yang bertambah besar hanya box luarnya saja bukan mapnya. Berikut adalah perbandingannya

Leaflet Contoh:
![leaflet contoh](https://user-images.githubusercontent.com/107049922/183310052-71f8344f-10ea-4b71-a608-7cd0e68429ee.png)

Leaflet Saya:
![leaflet saya](https://user-images.githubusercontent.com/107049922/183310064-eca13195-e163-4bd6-ab77-7769a65c42ae.png)

Terima Kasih!

**Comment by fafilia** [comment, marked as answer]
Halo @RanggaGemilang, untuk mengatur height leafleat bisa menggunakan parameter height saat merender output plot pada UI seperti contoh berikut:

```
box(width = 12,
            solidHeader = T,
            h3(tags$b("Ecological Status of Each Country")),
            leafletOutput("leaflet", height = 530))
```

Jika berkenan, bolehkah dilampirkan bagaimana code yang telah Bapak lakukan untuk mengatur height pada leaflet?

Adapun cara/alternatif lain yang bisa dilakukan, Bapak bisa mencoba salah satu cara yang terlampir dalam link berikut: https://stackoverflow.com/questions/36469631/how-to-get-leaflet-for-r-use-100-of-shiny-dashboard-height

**Reply by RanggaGemilang** [reply]
sudah bisa mba fafil dengan cara yang berikan diatas. Saya selama ini ngutak ngatik height box nya bukan leafletnya makanya tidak bisa hehe. Terima kasih!
